tdrs-backend/tdpservice/settings/common.py

```diff
@@ -67,6 +67,8 @@ class Common(Configuration):
     """Define configuration class."""
 
     INSTALLED_APPS = (
+        "colorfield",
+        "admin_interface",
```

Comment:
The mere addition of our `django-admin-508` package will supersede the default Django Admin templates.